com.spotify.EdgeClient.appdata.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.spotify.EdgeClient</id>
+  <provides><id>com.spotify.EdgeClient.desktop</id></provides>
+  <launchable type="desktop-id">com.spotify.EdgeClient.desktop</launchable>
+  <name>Spotify</name>
+  <developer_name>Spotify</developer_name>
+  <summary>Online music streaming service</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://www.spotify.com/us/legal/end-user-agreement/</project_license>
```

Comment:
is redistribution allowed?

Reply:
that doesn't answer the question, it's just you speculating.

Reply:
@hfiguiere I agree with you, I can contact Spotify and hope for an answer if it is preferred. However, not to be critical, but if this app can´t belong to Flathub then also https://github.com/flathub/com.spotify.Client/ should not.